spec/openapi.yaml

```diff
@@ -10125,6 +10484,92 @@ components:
         - termUids
         - authorization
       type: object
+
+    AttachmentResponse:
+      type: object
+      required:
+        - response
+      properties:
+        response:
+          allOf:
+          - $ref: '#/components/schemas/SuccessResponse'
+          - properties:
+              data:
+                $ref: '#/components/schemas/AttachmentResponseItem'
+            type: object
+
+    AttachmentLinkResponse:
+      properties:
+        response:
+          allOf:
+            - $ref: '#/components/schemas/SuccessResponse'
+            - properties:
+                data:
+                  properties:
+                    items:
+                      description: Source terms
+                      items:
+                        $ref: '#/components/schemas/AttachmentResponseItem'
+                      type: array
+                    totalCount:
+                      description: total number of attachemnts
```

Comment:
any limit?

Reply:
no paging there. it just duplicates number of array items